config-connector/solutions/README.md

```diff
@@ -0,0 +1,29 @@
+# Config Connector Solutions
+
+## Overview
+
+Config Connector Solutions provides best practice solutions
+to common cloud applications, formatted as YAML definitions
+for Config Connector CRDs. These YAMLs can be applied to
+clusters running [Config
+Connector](https://cloud.google.com/config-connector/docs/how-to/getting-started).
+
+## Structure
+
+Folders under this directory denote general solution areas.
+In each solution area folder, there are folders for each package
+manager (currently only kpt), under which are nested all available solutions in
```

Comment:
should we say "for each packing/customization tool" instead of "package manager"?

Reply:
I agree that 'package manager' might be a little misleading. I don't think 'customization tool' is quite right either though, and it's easy to conflate with kustomize.

Reply:
I would say packaging & customization tool because kpt does provide customization as well.